tests/foreman/cli/test_computeresource_osp.py

```diff
@@ -0,0 +1,274 @@
+# -*- encoding: utf-8 -*-
+"""
+:Requirement: Computeresource
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: CLI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import gen_string
+
+from robottelo.cli.computeresource import ComputeResource
+from robottelo.cli.factory import (
+    CLIFactoryError,
+    CLIReturnCodeError,
+    make_compute_resource
+)
+from robottelo.config import settings
+from robottelo.decorators import (
+    run_only_on,
+    stubbed,
+    skip_if_not_set,
+    tier1,
+    tier3,
+    upgrade
+)
+from robottelo.test import CLITestCase
+
+
+class OSPComputeResourceTestCase(CLITestCase):
+    """OSPComputeResource CLI tests."""
+
+    @classmethod
+    @skip_if_not_set('osp')
+    def setUpClass(cls):
+        super(OSPComputeResourceTestCase, cls).setUpClass()
+        cls.current_osp_url = settings.osp.hostname
+        cls.username = settings.osp.username
+        cls.password = settings.osp.password
+        cls.tenant = settings.osp.tenant
+
+    @tier1
+    @run_only_on('sat')
+    def test_positive_create_osp_with_valid_name(self):
+        """Create Compute Resource of type Openstack with valid name
+
+        :id: 064c939f-b2da-4b49-b784-24559f296cd9
+
+        :expectedresults: Compute resource is created
+
+        :CaseImportance: Critical
+        """
+        ComputeResource.create({
+            u'name': 'cr {0}'.format(gen_string(str_type='alpha')),
+            u'provider': 'Openstack',
+            u'user': self.username,
+            u'password': self.password,
+            u'tenant': self.tenant,
+            u'url': self.current_osp_url
+        })
```

Comment:
no assert?

Reply:
Think this should raise exception if the cmd is not successful, see the negative test

Reply:
think we can add  with self.assertNotRaises(CLIReturnCodeError) to have a clean assertion on create